sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1918,6 +1918,27 @@ object GpuOverrides {
         override def convertToGpu(): GpuExpression =
           GpuCreateNamedStruct(childExprs.map(_.convertToGpu()))
       }),
+    expr[ArrayContains](
+      "Returns a boolean if the array contains the passed in key",
+      ExprChecks.binaryProjectNotLambda(
+        TypeSig.BOOLEAN,
+        TypeSig.all,
```

Comment:
Does spark really output anything except boolean?

Reply:
My understanding of the checks is fairly stunted and I am really happy to receive this feedback as it helps me understand. Will update.